src/libs/antares/study/parts/hydro/container.cpp

```diff
@@ -174,9 +174,9 @@ bool PartHydro::LoadFromFolder(Study& study, const AnyString& folder)
 
             //	GUI part patch :
             //		We need to know, when estimating the RAM required by the solver, if the
-            //current
-            // area 		is hydro modulable. Therefore, reading the area's daily max power at this
-            // stage is 		necessary.
+            // current
```

Comment:
why is there change after clang ?

Reply:
Strange indeed, but my answer is the same as previouly.